### DISCUSSION TITLE
Website sent me to inappropiate websites

### DISCUSSION THREAD

**Discussion by UndarkAido** [Devolutions/UniGetUI#1806] [General]
I loaded https://www.marticliment.com/wingetui/ to see some screenshots and instead I got the front page of Chaturbate. It doesn't seem to happen regularly, I tried again and it's fine now, but you should really look into a different advertising platform. This is a fresh Windows and Firefox install, nothing else would have caused this to happen.

**Comment by UndarkAido** [comment]
Oh, if you delete cookies & site data it does it again with a different site

**Reply by Spacellary** [reply]
@UndarkAido - My friend, I actually think you got compromised by a malware that is hijacking your Browser.

This isn't happening here... 😅

Try scanning fully with [Malwarebytes](malwarebytes.com/mwb-download/).

Unless it an unintended intrusive Ad? If possible, please record a video.

**Reply by UndarkAido** [reply]
Fresh install of Windows, fresh install of Firefox through `winget`, only the fourth domain I visited after Microsoft, Google, and this GitHub repo. It occurs only on your page and it occurred every time I wiped your page's cookies and reloaded for the first four or so times until it stopped opening new pages and started just closing the page instead.

Edit: I'll try to figure out how to record a video, haven't done it in years.

**Reply by UndarkAido** [reply]
@Spacellary Here's a video showing the redirect behavior: https://youtu.be/K1eeDgpLESA
Running a Malwarebytes scan now, highly doubt it will find anything

**Reply by UndarkAido** [reply]
![image](https://github.com/marticliment/WingetUI/assets/7421794/cd84646a-0c1f-4b94-b920-2e28cf66aa42)
I got this after installing Malwarebytes then an extension to attempt to trace the redirect

**Comment by Spacellary** [comment]
I suppose this is something for @marticliment to address.

**Comment by marticliment** [comment]
Hello @UndarkAido. I am very sorry for the inconvenience.

I have checked the servers and they appear to be healthy. After a little bit of research, it looks as if the advertising platform was loading redirectors hidden on the banner ad scripts. I wil take down advertising immediately until I can get this fixed.
